am-lib/src/test/java/integration/uk/gov/hmcts/reform/amlib/FilterResourceIntegrationTest.java

```diff
@@ -48,21 +45,28 @@
     private static DefaultRoleSetupImportService importerService = initService(DefaultRoleSetupImportService.class);
     private String resourceId;
     private String accessorId;
+    private String roleName;
+    private String otherRoleName;
     private ResourceDefinition resourceDefinition;
 
     @BeforeEach
     void setUp() {
         resourceId = UUID.randomUUID().toString();
         accessorId = UUID.randomUUID().toString();
-        importerService.addResourceDefinition(
-            resourceDefinition = createResourceDefinition(serviceName, UUID.randomUUID().toString(), UUID.randomUUID().toString()));
+
+        importerService.addRole(roleName = UUID.randomUUID().toString(), IDAM, PUBLIC, AccessType.ROLE_BASED);
```

Comment:
`AccessType.ROLE_BASED` - either use static import or not, be consistent.

Reply:
👍